source/frontend/src/features/properties/map/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -67,76 +69,93 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
-              <SectionField
-                label="Expropriation notice served date"
-                labelWidth="4"
-                contentWidth="4"
-              >
-                <FastDatePicker
-                  field="expropriationNoticeServedDateTime"
-                  formikProps={formikProps}
-                />
-              </SectionField>
-              <SectionField label="Expropriation vesting date" labelWidth="4" contentWidth="4">
-                <FastDatePicker field="expropriationVestingDateTime" formikProps={formikProps} />
-              </SectionField>
-              <SectionField label="Special instructions" labelWidth="12">
-                <MediumTextArea field="specialInstruction" />
-              </SectionField>
-            </Section>
-            <Section header="Financial coding">
-              <SectionField label="Product" labelWidth={'4'}>
-                {acquisitionFile.product?.code ?? ''}
-              </SectionField>
-              <SectionField label="Business function" labelWidth={'4'}>
-                {acquisitionFile.project?.businessFunctionCode?.code ?? ''}
-              </SectionField>
-              <SectionField label="Work activity" labelWidth={'4'}>
-                {acquisitionFile.project?.workActivityCode?.code ?? ''}
-              </SectionField>
-              <SectionField label="Cost type" labelWidth={'4'}>
-                {acquisitionFile.project?.costTypeCode?.code ?? ''}
-              </SectionField>
-              <SectionField label="Fiscal year" labelWidth="4" contentWidth="4" required>
-                <Select field="fiscalYear" options={fiscalYearOptions} placeholder="Select..." />
-              </SectionField>
-            </Section>
-            <Section>
-              <SectionField label="Detailed remarks" labelWidth="12">
-                <MediumTextArea field="detailedRemarks" />
-              </SectionField>
-            </Section>
+              <Section header="Financial coding">
+                <SectionField label="Product" labelWidth="7">
+                  {acquisitionFile.product?.code ?? ''}
+                </SectionField>
+                <SectionField label="Business function" labelWidth="7">
+                  {acquisitionFile.project?.businessFunctionCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Work activity" labelWidth="7">
+                  {acquisitionFile.project?.workActivityCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Cost type" labelWidth="7">
+                  {acquisitionFile.project?.costTypeCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Fiscal year" labelWidth="7" contentWidth="4" required>
+                  <Select field="fiscalYear" options={fiscalYearOptions} placeholder="Select..." />
+                </SectionField>
+              </Section>
+
+              <Section header="Payment">
```

Comment:
The only problem I see with this approach is that payee is essentially untracked by formik, so non of the unsaved changes logic will work. That may result in this failing FT. 
This could be implemented using formik could it not? and then all of the normal form logic would work properly?

Reply:
not sure what you mean. Formik-wise, it only cares about the payee id+type combination. If that changes it will detect if the form needs updating

Reply:
nevermind, not sure what I was on about, you are right.